docs/Node/ReadLine.md

```diff
@@ -50,33 +66,33 @@ Prompt the user for input on the specified `Interface`.
 #### `setPrompt`
 
 ``` purescript
-setPrompt :: forall eff. String -> Int -> Interface -> Eff (console :: CONSOLE | eff) Interface
+setPrompt :: forall eff. String -> Int -> Interface -> Eff eff Interface
```

Comment:
If we choose the `eff` variable here to be `()`, then the return type is a pure eff computation, which means we can 'run' it with [`runPure`](http://pursuit.purescript.org/packages/purescript-eff/0.1.2/docs/Control.Monad.Eff#v:runPure), which is probably not what we want. Same for `close` and `setLineHandler`, come to think of it.

One way to prevent this could be to declare a new effect type in this module, just to make sure the rows in these functions aren't empty. `READLINE`, perhaps? Alternatively, if you have any other ideas, I'd be interested to hear them too.